eva/parser/load_statement.py

```diff
@@ -44,7 +44,7 @@ def __init__(
         self._file_options = file_options
 
     def __str__(self) -> str:
-        print_str = "LOAD FILE {} INTO {}({}) WITH {}".format(
+        print_str = "LOAD {} INTO {}({}) WITH {}".format(
```

Comment:
1. Should remove the `WITH` option here?
2. Is `self._file_options` used after this PR?

Reply:
1. Sure, I will fix this.
2. I still kept the `file_options`so that in the future if we plan to support more functionality, we can easily extend it.